### ISSUE TITLE
Frontiers: middle initials missing after first one, in bibliography

### ISSUE THREAD

**Issue by SimonDedman** [citation-style-language/styles#6488]
In my Bibtex entry I have:
`Doe, John AB` (e.g. Andrew Brian)
but when this is generated into a bibliography with Qiqqa's InCite, using the [Frontiers CSL](https://github.com/citation-style-language/styles/blob/master/frontiers.csl), it results in:
`Doe, J. A.`

My coauthor is keen that his middle initials are both present. I can do this manually but it seems like it should happen automatically? Can anyone work out where this issue arises? Thanks in advance!

**Comment by bwiernik** [comment]
This is an issue with the version of citeproc-js that Qiqqa InCite has installed. They need to update to a current version of citeproc-js.

**Comment by bwiernik** [comment]
You should store the name as Doe, John A. B.  (And in general, it is better to store data in CSL JSON or CSL YAML format when working with CSL styles, as conversion from BibTeX format is lossy and may not yield accurate results.)

**Comment by SimonDedman** [comment]
That works a treat, thanks for the quick reply Brenton.

Re your earlier message, if Qiqqa updates to a current version of `citeproc-js`, will my existing (default/automatic) Bibtex format `Doe, John AB` authors get populated as `Doe, J. A. B.` in future, do you think? Thanks again!

**Comment by bwiernik** [comment]
No, a bug some platforms have had when they used outdated versions of citeproc-js is middle initials not being shown at all even when entered correctly in the data. I had missed the way your data was entered on first read and thought that that was the issue here

**Comment by bwiernik** [comment]
In general, BibTeX data will not reliably produce citations with CSL styles, especially for non-book, non-journal items (eg, reports, software, media). I recommend using a more fully-specified data format like CSL-JSON, CSL YAML, or RIS when possible
